network/src/inbound/inbound.rs

```diff
@@ -262,32 +189,32 @@ impl<S: Storage + Send + Sync + 'static> Node<S> {
                     if sync.is_empty() {
                         // An empty `Sync` is unexpected, as `GetSync` requests are only
                         // sent to peers that declare a greater block height.
-                        self.peer_book.register_failure(source);
```

Comment:
if we no longer use this method and instead register peer failures using some alternative means, the method should be removed

Reply:
ok, I see it's removed, the diff was just collapsed